docs/conf.py

```diff
@@ -25,6 +25,11 @@
 copyright = "2018, Google"
 author = "Google APIs"
 
+# The full version, including alpha/beta/rc tags.
+release = google.cloud.ndb.__version__
+# The short X.Y version.
+version = ".".join(release.split(".")[:2])
```

Comment:
Is there a precedence for dropping patch precision?

Reply:
Good question. This was the code that was removed from conf.py when the hardcoded version variable was removed from __init__.py some time ago, and that's the only precedence I have. I suppose we're obviously not using it, since it was gone for months with no effects, so I think I'll leave it out of this commit.